llvm/test/CodeGen/AArch64/sve2p1-fdot.ll

```diff
@@ -0,0 +1,91 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 6
+; RUN: llc -mtriple=aarch64-linux-gnu -mattr=+sve2 < %s | FileCheck %s --check-prefixes=CHECK,SVE2
```

Comment:
If you add `target triple = "aarch64-linux-gnu"` to the ll file you can shorten all the RUN lines by removing the duplicate `-mtriple` options.